src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -158,6 +160,19 @@ class DataDownload extends AsyncView<Props, State> {
             <br />
             {this.renderDate(dateExpired)}
           </p>
+          <small>
+            <strong>SHA1:{checksum}</strong>
+          </small>
+          <p>
+            Need help verifying? Checkout our{' '}
```

Comment:
Noob question, does this need to be wrapped in a translation helper?

Reply:
Hm... good question, feels like it should though I'm not sure how to deal with the anchor tag within a translation helper. Maybe I should change it to a button and have 2 separate phrases? Ideas?

Reply:
@Zylphrex there's `tct` that covers this usecase

Reply:
It should. Good catch.